test/test_misc.F90

```diff
@@ -65,6 +66,19 @@ program test_misc
   mbay(1) = 1
   mbay(2) = 2
   if (iupb(mbay, 1, 1) .ne. 0) stop 11
+
+  ! Test gets1loc for YCEN and CENT positions
+  call gets1loc('YCEN', 4, isbyt, iwid, iret)
+  if ( iret .ne. -1 ) stop 12
+  call gets1loc('CENT', 4, isbyt, iwid, iret)
```

Comment:
IN all these cases we should be checking the values of all output parameters, not just iret. There's also:

```
C> @param[out] ISBYT   -- integer: Number of starting byte within Section 1
C>                        which contains value corresponding to S1MNEM
C> @param[out] IWID    -- integer: Width (in bits) of value corresponding
C>                        to S1MNEM, counting from the first bit of the
C>                        byte pointed to by ISBYT

```

Always test all the output parameters for every test.